org.eclipse.wb.core/src/org/eclipse/wb/internal/core/editor/structure/property/ComponentsPropertiesPage.java

```diff
@@ -80,25 +80,25 @@
   // IPage
   //
   ////////////////////////////////////////////////////////////////////////////
+  @Override
   public void dispose() {
     Control control = getControl();
     if (control != null && !control.isDisposed()) {
       control.dispose();
     }
   }
 
+  @Override

```

Comment:
I dislike mixing different concerns in one PR. Adding @override does not belong to the topic of this PR

Can you send another PR for adding the @overrides? You find clean-up action in the package explorer -> Right-mouse click -> Cleanup Missing Code tab. Please run this only with one cleanup action at the time to make review easier.